go_modules/lib/dependabot/go_modules/file_updater/go_mod_updater.rb

```diff
@@ -2,13 +2,32 @@
 
 require "dependabot/shared_helpers"
 require "dependabot/errors"
+require "dependabot/go_modules/file_parser"
 require "dependabot/go_modules/file_updater"
 require "dependabot/go_modules/native_helpers"
 
 module Dependabot
   module GoModules
     class FileUpdater
       class GoModUpdater
+        # Turn off the module proxy for now, as it's causing issues with
+        # private git dependencies
+        ENVIRONMENT = { "GOPRIVATE" => "*" }.freeze
+
+        RESOLVABILITY_ERROR_REGEXES = [
```

Comment:
These constants were below the `private` marker, which was confusing as constants are public in Ruby even if they follow the `private` marker.